tests/test_cost_calculator.py

```
@@ -0,0 +1,23 @@
+import pytest
+from translate_srt import TranslationDisplay
+
+
+def test_calculate_cost_valid_model():
```

Ruff (0.8.2)

Missing return type annotation for public function `test_calculate_cost_valid_model`

Add return type annotation: `None`

(ANN201)